docs/issue-reporting-and-lifecycle.md

```diff
@@ -18,46 +8,46 @@ issues of each type.
    reports to check if anybody else has reported the same thing. You can [scope
       searches by the label "bug"](https://github.com/hashicorp/terraform-provider-aws/issues?q=is%3Aopen+is%3Aissue+label%3Abug) to help narrow things down.
 
-- [ ] __Include steps to reproduce__: Provide steps to reproduce the issue,
+- __Include steps to reproduce__: Provide steps to reproduce the issue,
    along with your `.tf` files, with secrets removed, so we can try to
    reproduce it. Without this, it makes it much harder to fix the issue.
 
-- [ ] __For panics, include `crash.log`__: If you experienced a panic, please
+- __For panics, include `crash.log`__: If you experienced a panic, please
    create a [gist](https://gist.github.com) of the *entire* generated crash log
    for us to look at. Double check no sensitive items were in the log.
 
 ### [Feature Requests](https://github.com/hashicorp/terraform-provider-aws/issues/new?labels=enhancement&template=Feature_Request.md)
 
-- [ ] __Search for possible duplicate requests__: It's helpful to keep requests
+- __Search for possible duplicate requests__: It's helpful to keep requests
    consolidated to one thread, so do a quick search on existing requests to
    check if anybody else has reported the same thing. You can [scope searches by
       the label "enhancement"](https://github.com/hashicorp/terraform-provider-aws/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) to help narrow things down.
 
-- [ ] __Include a use case description__: In addition to describing the
+- __Include a use case description__: In addition to describing the
    behavior of the feature you'd like to see added, it's helpful to also lay
    out the reason why the feature would be important and how it would benefit
    Terraform users.
 
-### [Questions](https://github.com/hashicorp/terraform-provider-aws/issues/new?labels=question&template=Question.md)
+### Questions
 
-- [ ] __Search for answers in Terraform documentation__: We're happy to answer
+- __Search for answers in Terraform documentation__: We're happy to answer
    questions in GitHub Issues, but it helps reduce issue churn and maintainer
    workload if you work to [find answers to common questions in the
-   documentation](https://www.terraform.io/docs/providers/aws/index.html). Oftentimes Question issues result in documentation updates
+   documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs). Oftentimes Question issues result in documentation updates
    to help future users, so if you don't find an answer, you can give us
    pointers for where you'd expect to see it in the docs.
 
 ## Issue Lifecycle
 
-**Note:** For detailed information on how issues are prioritized, see the [prioritization guide](./prioritization.md).
+**Note:** For detailed information on how issues are prioritized, see the [prioritization guide](prioritization.md).
 
 1. The issue is reported.
 
```

Comment:
Add an item:

> 30 days after the issue has been closed it is locked, preventing further comments.

It would also be good to add an item or section about opening a new issue (rather than commenting on a closed issue or PR) if the problem has not been completely solved or causes a regression. [This comment](https://github.com/hashicorp/terraform-provider-aws/pull/17695#issuecomment-1150471187) is an example of one that had been lost in the noise.

Reply:
I've added to this section, and taken a note to look at the issue template as it might be worth adding that there too.